.gitignore

```diff
@@ -13,4 +13,7 @@ Gemfile.lock
 .rspec_status
 
 # tests output
+node_modules/
```

Comment:
should they be in `spec/*` ?

Reply:
`node_modules` folder is generated at root path. You mean change its location to `spec/node_modules`? How can we do this without change `yarn add` command?

Reply:
I thought that any of the now ignored directories/files (e.g. package.json) could have been created under `spec`, but if it's not the case ignore the comment